metadata-writer

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source google-chrome.info
+
+do_subst () {
+    sed -e "s|@@FULLDESC@@|${FULLDESC}|g" \
+        -e "s|@@MENUNAME@@|${MENUNAME}|g" \
+        -e "s|@@PACKAGE@@|${PACKAGE}|g" \
+        -e "s|@@PRODUCTURL@@|${PRODUCTURL}|g" \
+        -e "s|@@SHORTDESC@@|${SHORTDESC}|g" \
+        -e "s|/usr/bin/@@USR_BIN_SYMLINK_NAME@@|google-chrome-launcher|g" \
+        $1 > $2
+    echo "Generated $2 from $1"
+}
+
+install -d /app/share/applications
+install -d /app/share/appdata
+
+do_subst desktop.template /app/share/applications/google-chrome.desktop
+do_subst google-chrome.appdata.xml.template /app/share/appdata/google-chrome.appdata.xml
```

Comment:
I think we should do something to indicate that the wrapper isn't supported, and that unlike other Flatpak apps, is unconfined and has full host access.